clikit/src/main/java/org/adamalang/clikit/model/Group.java

```diff
@@ -0,0 +1,41 @@
+package org.adamalang.clikit.model;
+
+import org.adamalang.clikit.exceptions.XMLFormatException;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+public class Group {
+    /** A group is a representation of a collection of commands in the XML **/
```

Comment:
move to prior the "public class" as that will associate the comment with the class in javadoc